ClientCore/Enums/ClientTypeHelper.cs

```diff
@@ -1,15 +1,23 @@
 ﻿using System;
+using System.Linq;
+using ClientCore.Extensions;
 
 namespace ClientCore.Enums
 {
     public static class ClientTypeHelper
     {
+        private const ClientType ct = 0;
```

Comment:
Why defining this variable?

Reply:
> Why defining this variable?

It used to export names of enum options.